sig/coprocessor/membership.md

```diff
@@ -0,0 +1,26 @@
+# Coprocessor SIG Members
+
+## Active Members
+
+- [breeswish](https://github.com/breeswish): Tech Lead
```

Comment:
The "Tech Lead" is the title, so as maintainer, committer, reviewer, etc?

Reply:
How about moving to `Maintainers` section?